extensions/amp-story-interactive/0.1/amp-story-interactive-abstract.js

```diff
@@ -185,6 +225,13 @@ export class AmpStoryInteractive extends AMP.BaseElement {
 
     /** @protected {?../../amp-story/1.0/variable-service.AmpStoryVariableService} */
     this.variableService_ = null;
+
+    installHostCssOnce(
```

Comment:
I thought you were going to call this in `amp-story-interactive.js` before running the `registerElements` so it's guaranteed to be installed once, instead of trying to install it once per component.

Reply:
I was following the behavior of `instalStylesOrDoc`, which verifies whether the style has already been added. 

Subjectively, it makes more sense to me to place the call in the abstract implementation instead of `AMP.extension()`.   It's uniform with shadow styles, and it mostly corresponds to the classnames set in the same file.